### DISCUSSION TITLE
Add Other Uses when uploading the provide definition

### DISCUSSION THREAD

**Discussion by juMar2020** [SAP-samples/landscape-management-sample-scripts#19] [General]
Hi, How can I have Other Uses selected automatically when uploading the provider definition file? For example, if I upload the one you guys created for the job maintenance and then try to select Other Uses like Notifications. I am not able to save it. See the image attached. Note that I am using the standard edition and it makes sense to get that error. I was just wondering if there is another way of enabling this as part of the import or another way.
![lama-standard](https://user-images.githubusercontent.com/65096898/234378355-d9fa186e-9045-4c25-9154-be136ff3f4d5.png)

Thanks for your sharing all these scripts!

**Comment by branrld** [comment]
Hi,
the provider definition for job maintenance represents a remote function call (RFC) to an ABAP system. 
<img width="785" alt="image" src="https://user-images.githubusercontent.com/71727159/234546406-561fe566-9348-4ee7-b9bb-19c563468350.png">
 However, it's important to note that Remote Function Call provider definitions cannot be used for notifications, provisioning replacements, or other similar options. Different types of provider definitions would need to be used for those purposes, as shown in your screenshot.

If you import an XML file with preselected options, you can modify them before using them. For instance, if you forgot to set a specific option to "true" in the GUI, you can modify the XML file to set it to "true" instead. However, keep in mind that only the options allowed for that particular type of provider definition can be modified in this way.

I hope this answers your question a bit.
BR

**Reply by juMar2020** [reply]
Thanks BR! Unfortunately I could not see that as I dont have the enterprise version. Appreciate your response.

**Reply by sasisaplama** [reply]
Where we need to upload the Job_Maintenance.xml in LaMa?
Inside of operations.d folder ? And do we need to change anything in XML , I can see some ip address in XML file
